server/utils.js

```diff
@@ -78,3 +87,22 @@ utils.decodeSpecialChar = function(url) {
   url = url.replace(/%3F/g, '?');
   return url;
 };
+
+/**
+ * Generates session ID.
+ * @return {string} Session ID.
+ */
+utils.randomString = function() {
+  return Math.random().toString(utils.RANDOM_STRING_RADIX)
+    .substr(utils.RANDOM_STRING_START_INDEX, utils.RANDOM_STRING_LENGTH);
+};
+
+/**
+ * Validates a string with a regex.
+ * @param {string} str
+ * @param {RegExp} regex
```

Comment:
You might be misunderstanding the compiler. But anyway you are in fact using RegExp as commented below, so that you don't need the RegExp() conversion.